sql/core/src/main/java/org/apache/spark/sql/vectorized/ArrowColumnVector.java

```diff
@@ -414,6 +416,21 @@ final int getInt(int rowId) {
     }
   }
 
+  private static class DateMilliAccessor extends ArrowVectorAccessor {
+
+    private final DateMilliVector accessor;
+
+    DateMilliAccessor(DateMilliVector vector) {
+      super(vector);
+      this.accessor = vector;
+    }
+
+    @Override
+    final long getLong(int rowId) {
```

Comment:
We should use `getInt()` to represent the number of days from 1970-01-01 if we map the type to date type.